Add site-specific docs (#563)
EGI-Federation/documentation: content/en/users/service-providers/_index.md

---

> title: Site-specific documentation for Service Providers

Reviewer: ```suggestion
title: Service provider specific documentation for Users
```

linkTitle: Service provider specific documentation
type: docs
weight: 100
description: >

>   Documentation provided by each cloud provider in the EGI Federation.

Reviewer: ```suggestion
  Documentation specific to the individual services providers.
```
Or something around this idea but better phrased.

---

> Some cloud providers in the EGI Federation publish documentation about how

Reviewer: ```suggestion
Some service providers in the EGI Federation publish documentation about how
```

to work with their site. Below is a list:

- [CESNET-MCC](https://docs.cloud.muni.cz/)
- [IFCA-LCG2](https://confluence.ifca.es/display/IC/Computing+Resources)
- [IN2P3-IRES](https://scigne.fr/en/user-support/online-documentation/)

> This information is handy to deal with the particularities of each OpenStack
> deployment.

Reviewer: ```suggestion
This information is handy to deal with the particularities of each services deployed by a given provider. It usually covers OpenStack deployments, but may also document other services.
```
